haystack/modeling/training/base.py

```diff
@@ -1,9 +1,6 @@
 from typing import Optional, Union, Tuple, List, Callable
 
-from typing import TYPE_CHECKING
```

Comment:
Could you please explain why we got rid of these lines so that I understand a bit better? `_LRScheduler`is A response to this comment would be fine. :)

Reply:
The type hint for DistillationTrainer turned out to be wrong. Because of that I don't need to import TYPE_CHECKING anymore as it's just necessary for preventing the circular import of FARMReader. There was never really a reason to also use that for `_LRScheduler` so `_LRScheduler can just be imported normally.